src/test/resources/wiremock/__files/empty-ps-result.txt

```diff
@@ -1,7 +1 @@
 Type d'identifiant PP|Identifiant PP|Identification nationale PP|Nom de famille|Prénoms|Date de naissance|Code commune de naissance|Code pays de naissance|Lieu de naissance|Code sexe|Téléphone (coord. correspondance)|Adresse e-mail (coord. correspondance)|Code civilité|Code profession|Code catégorie professionnelle|Code civilité d'exercice|Nom d'exercice|Prénom d'exercice|Code type savoir-faire|Code savoir-faire|Code mode exercice|Code secteur d'activité|Code section tableau pharmaciens|Code rôle|Numéro SIRET site|Numéro SIREN site|Numéro FINESS site|Numéro FINESS établissement juridique|Identifiant technique de la structure|Raison sociale site|Enseigne commerciale site|Complément destinataire (coord. structure)|Complément point géographique (coord. structure)|Numéro Voie (coord. structure)|Indice répétition voie (coord. structure)|Code type de voie (coord. structure)|Libellé Voie (coord. structure)|Mention distribution (coord. structure)|Bureau cedex (coord. structure)|Code postal (coord. structure)|Code commune (coord. structure)|Code pays (coord. structure)|Téléphone (coord. structure)|Téléphone 2 (coord. structure)|Télécopie (coord. structure)|Adresse e-mail (coord. structure)|Code département (coord. structure)|Ancien identifiant de la structure|Autorité d'enregistrement|Autres identifiants|Code genre d'activité|
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
-||||''|||||||||||||||||||||||||||||||||||||||||||||||
```

Comment:
The previous file was very odd. The service has no reason to generate an extract with 7 lines and separators from an empty response !